javascript/javascript1/week2/student-manager.js

```diff
@@ -0,0 +1,42 @@
+//global variable declaration
+const class07Students = [];
+let index = 0;
+//function to add student to the array and check for class size, duplicate name check and the queen name
+function addStudentToClass(studentName) {
+  let count;
+  if (index < 6 || studentName === 'Margrethe') {
+    for (count = 0; count < index; count++) {
+      if (class07Students[count] === studentName) {
+        console.log('Student ' + studentName + ' is already in the class');
+        return;
```

Comment:
Your code is almost working fine, but I would like you to rewrite this question again with a different structure according to what is required.
Can you read the question again and use the next structure and hints
```js
const class07Students = [];
function addStudentToClass(studentName) {
    // code can go here...
    if (class07Students.length > 6) {
      console.log('Cannot add more students to class 07');
    }

    if (/*student exists*/) {
      console.log('Student Benjamin is already in the class');
    }

    if (studentName === 'queen') {
      // remove everyone and add her only
    }

    if (/* empty string*/) {
      // don't add
    }
}

function getNumberOfStudents() {
    // use array length
}
```
**Don't forget to test your code at**
https://codepen.io/hackyourfuture-cph/pen/PLPmOB
pase your javascript code at the top of JS
then the user interface should works.

Keep in mind the word `queen` is used not `Margrethe`